docs/index.rst

```diff
@@ -1365,13 +1387,397 @@ to setup alerts from there.
+
+   .. code:: shell
+
+      ccloud api-key create --resource cloud -o json
+
+#. Verify your output resembles:
+
+   .. code-block:: text
+
+      {
+         "key": "QX7X4VA4DFJTTOIA",
+         "secret": "fjcDDyr0Nm84zZr77ku/AQqCKQOOmb35Ql68HQnb60VuU+xLKiu/n2UNQ0WYXp/D"
+      }
+
+   The value of the API key, in this case ``QX7X4VA4DFJTTOIA``, and API secret,
+   in this case ``fjcDDyr0Nm84zZr77ku/AQqCKQOOmb35Ql68HQnb60VuU+xLKiu/n2UNQ0WYXp/D``,
+   will differ in your output.
+
+#. Set parameters to reference these credentials returned in the previous step.
+
+   .. code-block:: text
+
+      METRICS_API_KEY='QX7X4VA4DFJTTOIA'
+      METRICS_API_SECRET='fjcDDyr0Nm84zZr77ku/AQqCKQOOmb35Ql68HQnb60VuU+xLKiu/n2UNQ0WYXp/D'
+
+#. :ref:`Dynamically configure <kafka-dynamic-configurations>` the ``cp-demo`` cluster to use the Telemetry Reporter, which sends metrics to |ccloud|. This requires setting 3 configuration parameters: ``confluent.telemetry.enabled=true``, ``confluent.telemetry.api.key``, and ``confluent.telemetry.api.secret``.
+
+   .. code-block:: text
+
+      docker-compose exec kafka1 kafka-configs \
+        --bootstrap-server kafka1:12091 \
+        --alter \
+        --entity-type brokers \
+        --entity-default \
+        --add-config confluent.telemetry.enabled=true,confluent.telemetry.api.key=${METRICS_API_KEY},confluent.telemetry.api.secret=${METRICS_API_SECRET}
+
+#. Check the broker logs to verify the brokers were dynamically configured.
+
+   .. sourcecode:: bash
+
+      docker-compose logs kafka1 | grep confluent.telemetry.api.key
+
+   Your output should resemble the following, but the ``confluent.telemetry.api.key`` value will be different in your environment.
+
+   .. code-block:: text
+
+      kafka1            | 	confluent.telemetry.api.key = QX7X4VA4DFJTTOIA
+      kafka1            | 	confluent.telemetry.api.secret = [hidden]
+
+
+|crep|
+------
+
+#. If you have been running ``cp-demo`` for a long time, you may need to refresh your local token to log back into MDS:
+
+   .. sourcecode:: bash
+
+      ./scripts/helper/refresh_mds_login.sh
+
+#. Set the |crep| name.
+
+   .. code-block:: text
+
+      REPLICATOR_NAME=replicate-topic-to-ccloud
+
+#. Create a role binding to permit a new instance of |crep| to be submitted to the local connect cluster with id ``connect-cluster``.
+
+   Get the |ak| cluster ID:
+
+   .. literalinclude:: includes/get_kafka_cluster_id_from_host.sh
+
+   Create the role bindings:
+
+   .. code-block:: text
+
+      docker-compose exec tools bash -c "confluent iam rolebinding create \
+          --principal "User:connectorSubmitter" \
+          --role ResourceOwner \
+          --resource Connector:${REPLICATOR_NAME} \
+          --kafka-cluster-id ${KAFKA_CLUSTER_ID} \
+          --connect-cluster-id connect-cluster
+
```

Comment:
This command has an incomplete string cand can't be pasted and executed directly

Reply:
Fixed